lib/DDGC/Web/Controller/InstantAnswer.pm

```diff
@@ -17,10 +18,20 @@ use if debug, 'Data::Dumper';
 
 sub base :Chained('/base') :PathPart('ia') :CaptureArgs(0) {
     my ( $self, $c ) = @_;
+
+    my $last_modified = $c->d->rs('InstantAnswer')->last_modified;
+    $last_modified->formatter( 'DateTime::Format::HTTP' );
+
+    $c->response->header (
+        'Last-Modified' => $last_modified;
+    );
```

Comment:
The idea is to only generate and send new JSON when there has been a change, otherwise we will send cached data. especially for /repo/. We ought to be using a single endpoint for all four repos.